### DISCUSSION TITLE
personal-development-shahrivar-1403/

### DISCUSSION THREAD

**Discussion by giscus** [amir-pourmand/hugo-papermod-farsi-template#55] [Announcements]
# personal-development-shahrivar-1403/

ماه شهریور هم با تمام داستان‌های خودش به اتمام رسید و وقت اینه که گزارشم رو منتشر کنم.
قبل از اون بگم که این ماه کار چندانی نکردم و خداوند متعال با روحیات کمی سادیستیک که داره، دوست داشت باهام بازی کنه و اون بالا به تمسخر بشینه و هر و هر بخنده.
جدای از شوخی، این ماه کشفیات جالبی در مورد خودم کردم. از جمله این که من (و بعضی اعضای خانواده‌ام) زیاد آدم‌های نرمالی نیستیم و کله‌مون زیاد حرف می‌زنه و فکر می‌کنه و این می‌تونه داستان‌ساز بشه.

https://aprd.ir/personal-development-shahrivar-1403/

**Comment by masoudnasrollahi** [comment]
درود امیرخان. البته که متنت پر از کلیدواژه‌ها برای جست‌وجوی بیشتر و دنبال کردن مسیری هست که یاد گرفتی؛ اما کاش کار من را هم راحت می‌کردی. شاید مخاطب گزارشت فقط خودت باشی، نمی‌دونم؛ ولی این تمرکز اعجاب‌انگیز،‌ کنجکاوی و شکاکی و خلاصه این تیغ تیز تفکر، کاش در قلم هم جاری می‌شد و محتوایی تولید می‌کردی که از سر تا تهش راهنمای سلامتی و تغذیه برای هر کس باشه. و بعد با دونستن بنیاد و قاعده‌های بدن و مکانیزم‌هاش هر کس می‌تونست برنامۀ غذایی و رژیم خودش را بچینه. چون احتملا چندی بعد که سبک زندگی پایدارت را داشته باشی، دیگه حوصلۀ (جواب‌های طولانی به کامنت‌ها و) نوشتن محتوای ارزشمندتر را نداری!

**Reply by pourmand1376** [reply]
سلام مسعود جان. وقتت بخیر. 

راستش تو برنامه‌ام هست که کمی جلوتر که رفتم، چند تا پست بنویسم. یکی پست دوم در مذمت قند هست و دومی در ستایش چربی! شاید بعداً هم چند تا پست راجع به کتوژنیک و روزه مداوم و تحقیقات پشتش نوشتم. ولی اینها رو دوست دارم بگذارم برای مدتی دیگه که حداقل خودم نتایج خیلی خوبی از اینها گرفته باشم و دست خالی هم نباشم. 

ولی اون محتوای کامل و ارزشمندی هم که نظر شماست بعید می‌دونم من بتونم بنویسم. اون رو کسی می‌تونه بنویسه که در این حوزه «صاحب نظر» باشه. 

من نهایتاً بتونم تجربیات خودم از تغییر سبک غذایی و چهار تا مقاله و فیلم که این طرف اون طرف دیدم رو بیام تعریف کنم و خلاص. 

ولی ممنون که گفتی. کامنتت انگیزه‌ای شد که حتماً همین الان در طول مسیر و بعداً هم این‌ها رو مکتوب کنم. شاید به درد بخوره. شاید هم نه!

**Comment by masoudnasrollahi** [comment]
وقتت بخیر 🌱
https://t.me/Khorshid_Translation_Group/105
این فیلم را دوستم فرستاد و گفت که تأثیر عمیقی روش گذاشته و وحشتناک متحولش کرده. از همین استاد Robert Lustig است. من هم تا حالا چندین بار همین چند دقیقه را دیده‌‌ام

**Reply by pourmand1376** [reply]
سلام مسعود. 
ممنون از معرفی. علاقه خاصی به استاد دارم. اینه که ویدئوی کاملش رو پیدا کردم و به زودی کامل می‌بینم.

https://www.youtube.com/watch?v=EKkUtrL6B18
